src/pages/Book/BookedPlace.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import variables from '../../styles/variables';
+
+const dateToString = date => {
+  const dateArray = date.split('-');
+  return `${dateArray[0]}년 ${dateArray[1]}월 ${dateArray[2]}일`;
+};
```

Comment:
how about 
```suggestion
const dateToString = date => {
  const [year, month, day] = date.split('-');
  return `${year}년 ${month}월 ${day}일`;
};
```

그리고 이 함수는 다른 팀원분들도 같이 사용할 수 있게 util 함수로 빼서 관리하는게 더 좋을 것 같아요!